### PR TITLE
Tests for derive address, attest utxo

### PR DESCRIPTION
**Pull request by cool-user**

### CODE REVIEW

File: packages/hw-app-ada/src/Ada.js [hunk 1]
```diff
@@ -32,6 +33,20 @@ const INVALID_PATH = 0x5001;
 const INVALID_PATH_LENGTH = 0x5002;
 const INDEX_NAN = 0x5003;
 
+const chunkResult = (response: Buffer, chunks) => {
```
Comment (ppershing) [inline packages/hw-app-ada/src/Ada.js]: How does this ensure the whole data is split?

Also `response` parameters seems to be a too specific name, cannot this just be a general function?

File: packages/hw-app-ada/src/Ada.js [hunk 2]
```diff
@@ -79,15 +100,25 @@ export default class Ada {
...
+    outputNumber: number,
+    amount: number,
+    hmac: string
+  }> {
     const P1_INIT = 0x01;
     const P1_CONTINUE = 0x02;
     const P2_UNUSED = 0x00;
 
-    const CHUNK_SIZE = 255;
+    const CHUNK_SIZE = 100;
```
Comment (ppershing) [inline packages/hw-app-ada/src/Ada.js]: Why are we changing this constant?

Comment (cool-user) [reply packages/hw-app-ada/src/Ada.js]: It stopped working at the time we fiddled with stream size. So I lowered it and it works again.

Comment (cool-user) [reply packages/hw-app-ada/src/Ada.js]: If necessary, I might try to find higher working value.

Comment (ppershing) [reply packages/hw-app-ada/src/Ada.js]: Hmm, it should not have stopped to work. What is the error we get if you raise it to 255? (Stream buf size should be set to 300 right now ...)

File: packages/hw-app-ada/src/Ada.js (hunk 1 above)
Comment (cool-user) [reply packages/hw-app-ada/src/Ada.js]: It does not, it chunks only what you want. Also renamed the function

File: packages/hw-app-ada/src/Ada.js (hunk 2 above)
Comment (cool-user) [reply packages/hw-app-ada/src/Ada.js]: Yep, it works. Sorry, must have been that I tried it with stream size 100 on Ledger and than associated it with new change (not with my change).

File: packages/hw-app-ada/src/Ada.js [hunk 3]
```diff
@@ -33,13 +33,13 @@ const INVALID_PATH = 0x5001;
 const INVALID_PATH_LENGTH = 0x5002;
 const INDEX_NAN = 0x5003;
 
-const chunkResult = (response: Buffer, chunks) => {
+const chunk = (array, chunks) => {
```
Comment (ppershing) [inline packages/hw-app-ada/src/Ada.js]: maybe chunkBy?

also array -> data (bufer is not an array in technical sense)
I am also still unconvinced that it won't check for end.

There are also more failure modes, what if we run out of te data too quickly?

In general, I believe we should be quite strict about chunking reponse

**Review by ppershing — APPROVED** [review]

**Pull request merged**